envoy/network/connection.h

```diff
@@ -344,6 +344,12 @@ class ServerConnection : public virtual Connection {
    */
   virtual void setTransportSocketConnectTimeout(std::chrono::milliseconds timeout,
                                                 Stats::Counter& timeout_stat) PURE;
+
+  /**
+   * Inject the data into connection.
+   * @param data the data inject into the connection.
+   */
+  virtual void injectData(Buffer::Instance& data) PURE;
```

Comment:
I don't think this is ever used. The data is passed to the constructor, which I think is better anyways.